config/settings.py

```diff
@@ -17,6 +18,8 @@
     DEBUG=(bool, True),
     SECRET_KEY=(str),
     ALLOWED_HOSTS=(str, '*'),
+    IFRC_GATES_ENVIRONMENT=(str, 'staging'),
```

Comment:
```suggestion
    IFRC_GATES_ENVIRONMENT=(str, 'development'),
```

Reply:
instead of IFRC_GATES_*, can we name it like RCCE_CS (for RCCE COLLECTIVE SERVICE) wherever possible ?

Reply:
instead of IFRC_GATES_*, can we name it like RCCE_CS (for RCCE COLLECTIVE SERVICE) wherever possible ?

Reply:
instead of IFRC_GATES_*, can we name it like RCCE_CS (for RCCE COLLECTIVE SERVICE) wherever possible ?